app/views/reductions.html

```diff
@@ -0,0 +1,105 @@
+{% extends "includes/layout.html" %} 
+
+{% block content %}
+<main id="content" role="main">
+    {% include "includes/breadcrumbs.html" %}
+    
+    {% include "includes/validation-error-messages.html" %} 
+    
+    {% include "includes/title-subtitle.html"%}
+    
+    {% include "includes/sub-nav.html" %} 
+    
+    {% if successText %}
+    <div class="message message--success">{{ successText }}</div>
+    {% endif %}
+
+    <a href="/offender-manager/{{ linkId }}/add-reduction" class="button">Add a reduction</a>
+    
+    <h2 class="heading-large">Active<span class="visuallyhidden"> reductions</span></h2>
+
+    <table class="sln-table-reductions">
+        <thead>
+            <tr>
+                <th id="active_type">Type</th>
+                <th id="active_hours">Hours</th>
+                <th id="active_start_date">Start date</th>
+                <th id="active_end_date">End date</th>
+                <th></th>
+            </tr>
+        </thead>
+
+        <tbody class="">
+            {% if newReduction %}
+            <tr>
+                <td headers="active_type">{{ newReduction.reason }}</td>
```

Comment:
This placing a new Reduction always in active? It could be scheduled? Is this part of the next story?

Reply:
Yes, we should address it for the view story.

Reply:
Yep, next story.